packages/perseus-core/src/parse-perseus-json/perseus-parsers/numeric-input-widget.ts

```diff
@@ -29,12 +32,41 @@ const parseMathFormat = enumeration(
     "pi",
 );
 
-const parseSimplify = enumeration(
-    "required",
-    "correct",
-    "enforced",
-    "optional",
-);
+export const parseSimplify = pipeParsers(
+    union(constant(null))
+        .or(constant(undefined))
+        .or(boolean)
+        .or(constant("required"))
+        .or(constant("correct"))
+        .or(constant("enforced"))
+        .or(constant("optional"))
+        .or(constant("accepted")).parser,
+).then(convert(deprecatedSimplifyValuesToRequired)).parser;
+
+function deprecatedSimplifyValuesToRequired(
+    simplify:
+        | "required"
+        | "correct"
+        | "enforced"
+        | "optional"
+        | "accepted"
+        | null
+        | undefined
+        | boolean,
+): PerseusNumericInputSimplify {
+    switch (simplify) {
+        case "enforced":
+        case "required":
+        case "optional":
+            return simplify;
+        // NOTE(benchristel): "accepted", "correct", true, false, undefined, and
+        // null are all treated the same as "required" during scoring, so we
+        // convert them to "required" here to preserve behavior. See the tests
+        // in score-numeric-input.test.ts
```

Comment:
Greatly appreciate this comment as it is very helpful to remember all of the silly values allowed for this field.